rgf/test.py

```diff
@@ -9,7 +9,7 @@
 from sklearn.metrics import log_loss, mean_squared_error
 from sklearn.utils.testing import assert_less, assert_almost_equal
 import numpy as np
-from rgf import RGFClassifier, RGFRegressor
+from .rgf import RGFClassifier, RGFRegressor
```

Comment:
On my environment (Python 2.7), this relative import is failed.
Could you explain why you use relative import?